compiler/rustc_codegen_cranelift/src/value_and_place.rs

```diff
@@ -974,8 +974,8 @@ pub(crate) fn assert_assignable<'tcx>(
                 }
             }
         }
-        (&ty::Coroutine(def_id_a, args_a, mov_a), &ty::Coroutine(def_id_b, args_b, mov_b))
-            if def_id_a == def_id_b && mov_a == mov_b =>
+        (&ty::Coroutine(def_id_a, args_a), &ty::Coroutine(def_id_b, args_b))
+            if def_id_a == def_id_b =>
```

Comment:
Is this safe?

Reply:
Yes. Movability is a property uniquely determined by def-id -- its inclusion here was redundant.